app/views/relatorio/_show.html.erb

```diff
@@ -0,0 +1,65 @@
+<div class="wrapper wrapper-content animated fadeInRight">
+	<div class="row">
+		<div class="col-lg-12">
+			<div class="ibox float-e-margins">
+				<div class="ibox-title">
+					<div class="pull-right actions">
+						<button id="button_search_participacao" class="btn btn-warning btn-rounded" data-toggle="modal" data-target="#modal_search">
+							<i class="fa fa-search"></i>
+						</button>
+					</div>
+				</div>
+				<div class="ibox-content">
+					<div class="dataTables_wrapper form-inline">
+						<table class="table table-striped table-bordered table-hover dataTables-example dataTable">
+							<thead>
+								<tr>
+									<th><%="Foto".to_s.mb_chars.upcase.to_s%></th>
+									<th><%="Nome".to_s.mb_chars.upcase.to_s%></th>
+									<th><%="Ações Confirmadas".to_s.mb_chars.upcase.to_s%></th>
+										<th><%="Ações Presentes".to_s.mb_chars.upcase.to_s%></th>
+									<th><%="Percentual de Comparecimento".to_s.mb_chars.upcase.to_s%></th>
+									<th><%="Opções".to_s%></th>
+								</tr>
+							</thead>
+							<tbody>
+								<% @usuario.each do |usuario| %>
+									<% if usuario.participacoes.first != nil %>
+									<tr>
+										<td><%= image_tag((usuario.foto.thumb.url))%></td>
+										<td><%= usuario.nome%></td>
+										<td>
+											<% @model.all.each do |participacao| %>
+													<% if participacao.usuario === usuario%>
+														<%= participacao.acao.nome %> - <%= participacao.acao.projeto.nome %>
+														 <br>
+													<% end %>
+											<% end %>
+										</td>
+									<td>
+										<% @model.all.each do |participacao| %>
+											<% if participacao.acao.state === "concluida"%>
+												<% if participacao.state === "presente"%>
+													<% if participacao.usuario === usuario%>
+														<%= participacao.acao.nome %> - <%= participacao.acao.projeto.nome %>
```

Comment:
esse if não ficaria melhor em uma linha apenas ? 

if participaçao.acao_state === "Concluida" and participacao.state === " " and participacao. usuario === usuario